src/Components/Search/SearchBar.tsx

```diff
@@ -169,26 +172,37 @@ export class SearchBar extends Component<Props, State> {
   }
 
   // Navigate to selected search item.
-  onSuggestionSelected = (
+  @track(
+    (
+      _props,
+      state: State,
+      [
+        _e,
+        {
+          suggestion: {
+            node: { href, searchableType, id },
+          },
+          suggestionIndex,
+        },
+      ]
+    ) => ({
+      action_type: Schema.ActionType.SelectedItemFromSearch,
+      query: state.term,
+      destination_path: href,
+      item_type: searchableType,
+      item_id: id,
+      item_number: suggestionIndex,
+    })
+  )
+  onSuggestionSelected(
     _e,
     {
       suggestion: {
         node: { href, searchableType, id },
       },
       suggestionIndex,
```

Comment:
You only still need the `href` right? So no need to destructure the rest (which actually has a tiny bit of runtime overhead, after being compiled).

Reply:
Good call!

Reply:
Lol actually I am using all of these in the track call 🙈

Reply:
Oh you mean in here, (where they are no longer used since its not being called imperatively), you're totally right.